SCSS/Imports/_museum.scss

```diff
@@ -17,6 +18,10 @@
     height: auto;
     margin:3%;
   }
+  footer{
+    background-color: $limeGreen;
+    height: 100px;
```

Comment:
Can you add a bit of space under the button, for me the button just sits flush against the bottom